repository-data/webfiles/src/main/resources/site/freemarker/hee/pages/minihub-main.ftl

```diff
@@ -0,0 +1,85 @@
+<#include "../../include/imports.ftl">
+<#include "../macros/guidance-content.ftl"/>
+
+<#-- @ftlvariable name="document" type="uk.nhs.hee.web.beans.MiniHub" -->
+<#-- @ftlvariable name="currentGuidance" type="uk.nhs.hee.web.beans.Guidance" -->
+<#-- @ftlvariable name="previousGuidance" type="uk.nhs.hee.web.beans.Guidance" -->
+<#-- @ftlvariable name="nextGuidance" type="uk.nhs.hee.web.beans.Guidance" -->
+<#if document??>
+    <div class="nhsuk-width-container">
+        <main id="maincontent" role="main" class="nhsuk-main-wrapper">
+
+            <div class="nhsuk-grid-row">
+                <div class="nhsuk-grid-column-two-thirds">
+                    <h1>
+                      <span role="text">${document.title}
+                        <span class="nhsuk-caption-xl nhsuk-caption--bottom">
+                          ${document.summary}
+                        </span>
+                      </span>
+                    </h1>
+
+                    <nav class="nhsuk-contents-list" role="navigation" aria-label="Pages in this guide">
+                        <h2>Contents</h2>
+                        <ol class="nhsuk-contents-list__list">
+                            <#list document.guidances as guidance>
+                                <#if guidance == currentGuidance>
+                                    <li class="nhsuk-contents-list__item" aria-current="page">
+                                        <span class="nhsuk-contents-list__current">${guidance.title}</span>
+                                    </li>
+                                <#else>
+                                    <li class="nhsuk-contents-list__item">
+                                        <a class="nhsuk-contents-list__link"
+                                           href="${guidance.name}">${guidance.title}</a>
+                                    </li>
+                                </#if>
+                            </#list>
+                        </ol>
+                    </nav>
+                </div>
+            </div>
+
+            <article>
+                <#if currentGuidance??>
+                    <@guidance guidanceDocument=currentGuidance/>
+                </#if>
+                <nav class="nhsuk-pagination" role="navigation" aria-label="Pagination">
+                    <ul class="nhsuk-list nhsuk-pagination__list">
+                        <#if previousGuidance??>
+                            <li class="nhsuk-pagination-item--previous">
+                                <a class="nhsuk-pagination__link nhsuk-pagination__link--prev"
+                                   href="${previousGuidance.name}">
+                                    <span class="nhsuk-pagination__title">Previous</span>
```

Comment:
@thojava - Trivial one, could we manage `Previous` & `Next` as Global resource bundles (if that makes senes) ?

Reply:
@smanivasag Yeap I just make it quickly as WIP so have not making it perfect yet :)